tutorials/Atmos/densitycurrent.jl

```diff
@@ -0,0 +1,367 @@
+# # [Density Current](@id EX-RTB-docs)
+#
+# In this example, we demonstrate the usage of the `ClimateMachine`
+# [AtmosModel](@ref AtmosModel-docs) machinery to solve the fluid
+# dynamics of the clasical density current test by Straka 1993.
+# We solve a flow in a
+# [`FlatOrientation`](@ref LESConfig) (Box) configuration - this is
```

Comment:
Can we change the `FlatOrientation` ref here? Not sure why it points to the LESConfig.

Reply:
> Can we change the `FlatOrientation` ref here? Not sure why it points to the LESConfig.

sure, what should I change it to. LESConfig uses FlatOrientation. I guess that's why it's there. All of these comments come from previously defined tutorials.

Reply:
Yes, but that's probably not the best reference to use. I'd suggest making `FlatOrientation` point to `FlatOrientation` and `LESConfig` point to `LESConfig`. So, let's use whichever reference we want to direct the user to.